desktop/app/index.js

```diff
@@ -11,6 +11,14 @@ import storeHelper from './store-helper'
 import mainWindow from './main-window'
 import helpHelper from './help-helper'
 
+// MUST do this else we get limited by simultaneous hot reload event streams
+app.commandLine.appendSwitch('ignore-connections-limit', 'localhost')
```

Comment:
this was causing the issue. we'd have all these hot reload event streams staying open which starved our connections to localhost